### DISCUSSION TITLE
Can I hide the title in the standard titlebar? (Tauri v2)

### DISCUSSION THREAD

**Discussion by timwehrle** [tauri-apps/tauri#10202] [Q&A]
So basically title. I created my own titlebar but I kinda didn't see a sense in it because I'm just rebuilding the standard titlebar. But is there an option to hide the title in the standard titlebar?

**Comment by fynntang** [comment]
Try adding `app.windows[0].titleBarStyle: "Overlay"` in `tauri.conf.json`, after adding this will cause the window can not drag, you need to add `data-tauri-drag-region` to the custom `titleBar`, To view detail [data-tauri-drag-region](https://v2.tauri.app/plugin/window-customization/#html)

**Reply by timwehrle** [reply]
And this hides the title of the titlebar? Or what does this do?
